packages/core/src/quote/Quote.ts

```diff
@@ -41,26 +40,21 @@ import { QuoteSchema } from './QuoteSchema'
  */
 
 export function validateQuoteSchema(
-  schema: unknown,
+  schema: JsonSchema.Schema,
   validate: unknown,
   messages?: string[]
 ): boolean {
-  const ajv = new Ajv()
-  ajv.addMetaSchema(QuoteSchema)
-  const result = ajv.validate(
-    JSON.parse(JSON.stringify(schema)),
-    JSON.parse(JSON.stringify(validate))
-  )
-  if (!result && ajv.errors) {
-    if (messages) {
-      ajv.errors.forEach((error: Ajv.ErrorObject) => {
-        if (typeof error.message === 'string') {
-          messages.push(error.message)
-        }
-      })
-    }
+  const validator = new JsonSchema.Validator(schema)
+  if (schema.$id !== QuoteSchema.$id) {
+    validator.addSchema(QuoteSchema)
+  }
+  const result = validator.validate(validate)
+  if (!result.valid && messages) {
+    result.errors.forEach((error) => {
```

Comment:
Probably out of scope for this PR, but I am not sure what is the purpose of the input `messages` parameter, as, as far as I have seen, it is not used anywhere in the codebase and it is not tested in any test scenario

Reply:
Yeah, this is for finding out, what exactly went wrong while verifying an object.
Not sure what to do about it, since it could be useful for people, if e.g. they want to inform a user filling out a form, what did go wrong.
I guess tests should be at least added in the future.